core/symbol/casecmp_spec.rb

```diff
@@ -1,6 +1,16 @@
 # -*- encoding: utf-8 -*-
 require File.expand_path('../../../spec_helper', __FILE__)
 
+LATIN_UPPER_A_TILDE = "\xC3".b.to_sym
```

Comment:
I've refactored the symbols into constants since we were duplicating these across several tests

Reply:
While this is a generally good idea, in this case adding top-level constants could potentially be problematic due to conflicts. Also, we try to avoid running too much code outside of examples in ruby/spec, so if say `.b` or `.to_sym` raises, the specs not relying on those values can still be run.

I am sorry to ask this, but could you keep them as locals? I prefer a bit of duplication to not being able to run specs due to code running outside of examples.
(In other situations, it might be nice to use `before :each` and instance variables, but this would have the same problem here)

Reply:
No problem, I'll make that change now :+1: